CMakeLists.txt

```diff
@@ -564,6 +560,14 @@ if (ENABLE_ALL OR ENABLE_AZURE)
 	createExtension(AZURE-EXTENSIONS "AZURE EXTENSIONS" "This enables Azure support" "extensions/azure" "${TEST_DIR}/azure-tests")
 endif()
 
+## Add the systemd extension
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+	option(DISABLE_SYSTEMD "Disables the systemd extension." OFF)
```

Comment:
The reason it's `DISABLE_SYSTEMD` is that it's enabled by default on supported platforms and I think the pattern is that the option is the opposite of the default. There's already an exception to this, but it might be an accident or just that it doesn't correspont to an extension (AFAIK): `ENABLE_OPS`.
Given this, do you still want to change it?

Reply:
changed in f1cd129